overlord/snapstate/snapmgr.go

```diff
@@ -254,7 +256,13 @@ func readInfo(name string, si *snap.SideInfo, flags int) (*snap.Info, error) {
 		if si != nil {
 			info.SideInfo = *si
 		}
-		return info, nil
+		err = nil
+	}
+	if err == nil && flags&loadExtra != 0 {
+		e := backend.LoadExtraInfo(info)
+		if e != nil && e != backend.ErrNoExtraInfo {
+			logger.Debugf("cannot read extra snap info for %q: %v", name, e)
```

Comment:
This is a nitpick but I think we should not log this unless we were expecting the snap to have this information. If we could store a flag in the state that extra info is available in a separate file this would be a natural and clear indication that something is wrong. Without this flag this is just spam that people learn to ignore (if it happens for unexpected reasons that would signify a real problem).

Reply:
nah, not worth the flag on state I don't think. We only log if the file existed and we couldn't read it.